_includes/head.html

```diff
@@ -3,7 +3,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
   
     <!-- Icone du site -->
-    <link rel="icon" href="{{"assets/img/logo.png" |relative_url}}" />
+    <link rel="icon" href="{{"assets/img/logo-1.png" |relative_url}}" />
```

Comment:
add / in config.yml or '/assets'

Reply:
slash (/) is already in baseurl so for make it conventionally, everybody calls object path without slash infront, for example: href="{{" assets/img/logo.png" | relative_ur l}}". (This convention will be updated in wiki section also).